Notes from the june 23 2026 ec meeting (#184)
jupyter-governance/ec-team-compass: docs/meeting_notes/2026.md

    * Jason is currently dedicating approximately 10 hours per week as a part-time Jupyter Foundation community manager, focusing on member retention, executive director recruitment, and organizing the upcoming all-member meeting.
* Contributor Recognition Programs
    * The Distinguished Contributors Award program, which provides $500 per recipient from charity funds, is under strategic review to adapt to a smaller contributor pipeline, with potential changes due by year-end.
    * Rus, Rick, and Darian noted that the contributor awards provide crucial external recognition for maintainers whose open-source work isn't fully acknowledged by their employers, and Rus plans to leverage marketing to track its impact on recruitment.
* Foundation vs. Community Dynamics
    * Project leaders are pushing to clarify the boundaries between volunteer community efforts and foundation-supported tasks, ensuring that non-technical operations like marketing and communications have clear ownership and a unified voice.

>     * A structural misalignment exists between the Linux Foundation's focus on user base growth and the community's focus on safeguarding operations, though Jason noted the incoming Executive Director will focus heavily on fundraising to secure long-term sustainability.

Reviewer: ```suggestion
    * A structural misalignment exists between the Linux Foundation's focus on user base growth and the community's focus on safeguarding operations, though Jason noted that when an Executive Director is hired, that person will focus on fundraising to secure long-term sustainability.
```

    * Jason is currently dedicating approximately 10 hours per week as a part-time Jupyter Foundation community manager, focusing on member retention, executive director recruitment, and organizing the upcoming all-member meeting.
* Contributor Recognition Programs
    * The Distinguished Contributors Award program, which provides $500 per recipient from charity funds, is under strategic review to adapt to a smaller contributor pipeline, with potential changes due by year-end.
    * Rus, Rick, and Darian noted that the contributor awards provide crucial external recognition for maintainers whose open-source work isn't fully acknowledged by their employers, and Rus plans to leverage marketing to track its impact on recruitment.
* Foundation vs. Community Dynamics
    * Project leaders are pushing to clarify the boundaries between volunteer community efforts and foundation-supported tasks, ensuring that non-technical operations like marketing and communications have clear ownership and a unified voice.

>     * A structural misalignment exists between the Linux Foundation's focus on user base growth and the community's focus on safeguarding operations, though Jason noted the incoming Executive Director will focus heavily on fundraising to secure long-term sustainability.

Reviewer: "the incoming Executive Director" -> this reads strangely to me. We don't have an incoming executive director do we?

Author: How about, "...when an Executive Director is hired, that person will focus on fundraising..."
